src/components/plots/linePlot.ts

```diff
@@ -172,6 +166,14 @@ export module Plots {
                      .defined((innerDatum, innerIndex) => definedProjector(innerDatum, innerIndex, dataset))(datum, index);
       };
     }
+
+    protected _getDataToDraw() {
```

Comment:
Because `Line` and `Area` have their own separate way of dealing with undefined data within their appropriate path generators